internal/utils/min_max_avx2_amd64.s

```diff
@@ -133,7 +133,7 @@ LBB0_5:
 	JNE  LBB0_9
 	JMP  LBB0_10
 
-TEXT ·_uint8_max_min_avx2(SB), $0-32
+TEXT ·_uint8_max_min_avx2(SB), NOSPLIT, $0-32
```

Comment:
I'm not familiar with assembly but is `$0` OK here? Should we use `$8` like others?